lib/standard/string.nit

```diff
@@ -1163,10 +1163,13 @@ class FlatString
 	#              String Specific Methods           #
 	##################################################
 
-	private init with_infos(items: NativeString, len: Int, from: Int, to: Int)
+	# Create a new string with given data.
+	#
+	# `items` will be used as is to retrieve the characters of the string.
```

Comment:
Proposal:

~~~nit
# `items` is used without copy, so that aliasing issues is the responsibility of the caller.
~~~

Yeah, that definitely looks better than the original version.